rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp.adoc

```diff
@@ -0,0 +1,7 @@
+= MuleSoft IDP 
```

Comment:
Is there any way to combine this with the other topic so you don't have a single 3rd-level bullet? Also, this topic is super short.